squad-server/plugins/discord-admin-request.js

```diff
@@ -123,9 +131,39 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
 
     await this.sendDiscordMessage(message);
 
-    await this.server.rcon.warn(
-      info.player.steamID,
-      `An admin has been notified, please wait for us to get back to you.`
-    );
+    await this.server.updatePlayerList;
+    let admins = await this.server.getAdminsWithPermission('canseeadminchat');
+
+    if (this.options.warnInGameAdmins) {
+      for (const player of this.server.players) {
+        if (admins.includes(player.steamID)) {
+          this.amountAdmins++;
+
+          await this.server.rcon.warn(player.steamID, `[${info.player.name}] - ${info.message}`);
+        }
+      }
+
+      if (this.amountAdmins === 0) {
+        await this.server.rcon.warn(
+          info.player.steamID,
+          `There are no in-game admins, however an admin has been notified via discord, please wait for us to get back to you.`
+        );
+      } else {
+        await this.server.rcon.warn(
+          info.player.steamID,
+          `There ${this.amountAdmins > 1 ? 'are' : 'is'} ${this.amountAdmins} in-game admin${
+            this.amountAdmins > 1 ? 's' : ''
+          }. Please wait for us to get back to you.`
+        );
+      }
```

Comment:
I think it would be a good idea to split this into a separate option other than `warnInGameAdmins`. Some may want the warnings for in-game admins but without this because it makes it easier for someone to check whether there's a lack of admins and therefore whether it's a good time to troll.

Reply:
![image](https://user-images.githubusercontent.com/25463237/110180453-16aad800-7e0a-11eb-8952-b2f3f6ffb34f.png)